tests/conftest.py

```diff
@@ -72,11 +74,44 @@ def compile_and_run(device, reset_torch_dynamo, request):
             accuracy = calculate_accuracy(outputs, outputs_after)
             if accuracy:
                 comp_runtime_metrics["accuracy"] = accuracy
+
+            # Memory analysis
+            mm = option._memory_manager
+            compiled_memory_metric = {}
+            peak_usage = mm.peak_sram_usage / 1048576
```

Comment:
`1048576` what is this constant?

Reply:
1024 * 1024 = 1048576 (1 MB in bytes)
To get peak sram usage in MB.

Reply:
if I have to ask, its a good signal this should be some constant variable with a clear name, right?

Reply:
Don't forget to address this